### PR TITLE
Script as per interface

### PR DESCRIPTION
**Pull request by SeventhNadir**
Changed the behavior of readFile to now read a stream from stdin with the idea that it leads to easier composition on the commandline. 

Created run.sh as it seems to be the convention for other languages that satisfies the exercism test runner interface.

### CODE REVIEW

File: cmd/testoutput/main.go [hunk 1]
```diff
@@ -33,21 +33,35 @@ func main() {
 	fmt.Println(string(bts))
 }
 
-func readFile(file string) ([][]byte, error) {
-	bts, err := ioutil.ReadFile(file)
+func readStream() ([][]byte, error) {
+	_, err := os.Stdin.Stat()
 	if err != nil {
-		return nil, err
+		panic(err)
```
Comment (tehsphinx) [inline cmd/testoutput/main.go]: better return the error here and let the `main()` function do the `log.Panic(err)`.

File: cmd/testoutput/main.go [hunk 2]
```diff
@@ -33,21 +33,35 @@ func main() {
...
+
+	reader := bufio.NewReader(os.Stdin)
+	var output []byte
+
+	for {
+		input, err := reader.ReadByte()
+		if err != nil {
+			break
+		}
+		output = append(output, input)
```
Comment (tehsphinx) [inline cmd/testoutput/main.go]: Do you see any advantage of this over `ioutil.ReadAll`?

**Comment by tehsphinx** [comment]
@SeventhNadir: Nice thinking on turning this into a stream! I like it.

I left some notes on what I'd probably change. Feel free to discard the second one if you like this more.

File: cmd/testoutput/main.go (hunk 1 above)
Comment (SeventhNadir) [reply cmd/testoutput/main.go]: I didn't notice that's what it was doing, but that makes a lot more sense!

**Comment by SeventhNadir** [comment]
Good pointers tehsphinx, I think ioutil.ReadAll is probably the more appropriate tool for this job

**Pull request merged**